src/software/ai/hl/stp/play/defense_play_test.py

```diff
@@ -32,60 +32,60 @@
     ],
 )
 def test_defense_play(simulated_test_runner, yellow_positions):
```

Comment:
@itsarune The behaviour of this play seems a bit weird. One of the friendly robots is running shadow enemy and and almost blocking our attacker, and the attacker also chips the ball into the net which I don't think is allowed
[defense_play_test_chipping_and_positioning.zip](https://github.com/UBC-Thunderbots/Software/files/11069494/defense_play_test_chipping_and_positioning.zip)

Reply:
that's pretty funny, but DefensePlay is currently unused. Eventually william's changes will replace this